lib/src/front_end/chain_builder.dart

```diff
@@ -176,10 +187,8 @@ class ChainBuilder {
   ///     .baz[0][1]
   ///     .bang()
   void _unwrapCall(Expression expression) {
-    var isCascade = _root is CascadeExpression;
```

Comment:
Remove the `CascadeSection` comment above?

Reply:
Good catch, thanks!